syntax = "proto3";

package http2;

message ServerConfig {
    string uri = 1;
    uint32 thread_count = 2;
}
